apps/web/components/booking/BookingListItem.tsx

```diff
@@ -427,7 +418,9 @@ const RecurringBookingsTooltip = ({
                     : getEveryFreqFor({
                         t,
                         recurringEvent: booking.eventType.recurringEvent,
-                        recurringCount: booking.recurringBookings.length,
+                        recurringCount: recurringDates.filter((date) => {
+                          return date >= now;
+                        }).length,
```

Comment:
This is to just show the number of "active" instances in the tooltip trigger:
<img width="227" alt="image" src="https://user-images.githubusercontent.com/467258/194352220-4383d703-8928-40c9-81e4-46147a0b623e.png">